ricoconverter/ricoconverter-convert/src/test/resources/eac2rico/_22occupation/expected.xml

```diff
@@ -24,19 +24,20 @@
          <rico:PerformanceRelation rdf:about="performanceRelation/005205-d699msj02xm-19qbb7wdtto3j">
             <rico:performanceRelationHasTarget rdf:resource="agent/005205"/>
             <rico:performanceRelationHasSource>
-               <rico:Activity>
+               <rico:Activity rdf:nodeID="_Activity-a">
                   <rico:activityIsSourceOfPerformanceRelation rdf:resource="performanceRelation/005205-d699msj02xm-19qbb7wdtto3j"/>
                   <!-- note that the following assertion is possible only with RiC-O v0.2, where OccupationType has become a subclass of ActivityType-->
                   <rico:hasActivityType rdf:resource="occupationType/FRAN_RI_010-d699msj02xm-19qbb7wdtto3j"/>
                </rico:Activity>
             </rico:performanceRelationHasSource>
          </rico:PerformanceRelation>
       </rico:agentIsTargetOfPerformanceRelation>
+      <rico:performsOrPerformed rdf:nodeID="_Activity-a" />
```

Comment:
Hi @tfrancart, here IMHO the following line can be added (as the entity is a Person that performs an Activity that has type an OccupationType):
`<rico:hasOrHadOccupationOfType rdf:resource="occupationType/FRAN_RI_010-d699msj02xm-19qbb7wdtto3j"/>`
Same below line 52